src/Entity/Media.php

```diff
@@ -281,12 +283,24 @@ public function setFile(File $file = null): Media
             return $this;
         }
 
+        if ($this->isVideo()) {
```

Comment:
This [line](https://github.com/EnMarche/en-marche.fr/pull/1404/files#diff-68a5398926b02f9663cde4e3d436fe02R282) may be a problem to get here